code/__DEFINES/components.dm

```diff
@@ -155,6 +156,9 @@
 // /mob/living/carbon signals
 #define COMSIG_CARBON_SOUNDBANG "carbon_soundbang"					//from base of mob/living/carbon/soundbang_act(): (list(intensity))
 
+// /mob signals
+#define COMSIG_MOB_ITEM_ATTACK "mob_item_attack"				//from base of /obj/item/attack(): (mob/M, mob/user)
```

Comment:
This already exists. Are you sure you updated from master after you resolved conflicts from my component PR from being merged?